_compdemos/gizmo-node-beta.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Gizmo Node Beta
+main_author: Michael Gutteridge
+primary_reviewers: atombaby
+---
+
+This page describes the new gizmo cluster nodes, some of the differences
+between these new systems, and how you can help bring these nodes into service.
+
+# The Hardware
+
+These new systems will be the "K" class of nodes.  Currently we have four in
+service- F, G, H, and J.  These new systems have 36 cores[1](#notes-1) each
+running at a peak of 3.10GHz.  These have been configured with about 768GB of
+RAM and 6TB of node-local storage.
+
+> Each node is equipped with an NVIDIA GeForce RTX 2080 Ti GPU.  At this time
+> it is not currently enabled for use- this feature will come soon.
+
+Along with these new nodes there are three new Rhino-class hosts to provide interactive and login sessions. These have the same 36 physical cores, but hyperthreading is enabled which increases the number of cores presented to 72.
+
+# The Software
+
+The software stack is significantly different from what is currently on gizmo:
+
+ - The base OS has been upgraded to Ubuntu 18.04 (Bionic) from the
+   venerable 14.04 (Trusty) currently installed
+ - The applications in _app_ have been rebuilt for this new platform
+
+Our goal is to make the versions of the tools you're using on the current hosts
+available in these new nodes.  We've already compiled quite a few, but if
+you're missing one, email SciComp.
```

Comment:
Link to easybuild-life-sciences issues?

Reply:
Could do- I think typically we've gone through the ticketing system.  @fizwit what's your opinion on that?  Incidentally, maybe this would be a good place for a github issue template (i.e. software requests)

Reply:
Would this be the link: https://github.com/FredHutch/easybuild-life-sciences/issues